web/src/components/Home/index.tsx

```diff
@@ -1,12 +1,21 @@
 import React from 'react'
-import './index.css'
+import styled from 'theme'
+
+const HomeContainer = styled.div`
+  margin-top: 40px;
+`
+
+const HomeHeader = styled.div`
+  display: flex;
+  justify-content: center;
+`
```

Comment:
This is an artifact of the Razzle build tool, you can combine them into one style tag using the semantic container since it gives you automatic responsive padding 
```suggestion
const StyledContainer = styled(Container)`
  display: flex;
  justify-content: center;
`
```